IoTuring/Entity/Deployments/Disk/Disk.py

```diff
@@ -103,6 +121,68 @@ def Update(self) -> None:
             attributeKey=EXTRA_KEY_DISK_FREE,
             attributeValue=usage.free,
             valueFormatterOptions=VALUEFORMATOPTIONS_DISK_GB)
+        
+        if OsD.IsLinux():
+            # because of this reverse parsing to get the devicename from the partition name, all the following is only linux having the partition named nvme* or sd* 
+            if self.configuredIo:
+                devname = self.disk_partition.device.split("/", 2)
+                if "nvme" in self.disk_partition.device:
+                    disk_io = psutil.disk_io_counters(perdisk=True)[devname[2][:-2]]
+                elif "sd" in self.disk_partition.device:
+                    disk_io = psutil.disk_io_counters(perdisk=True)[devname[2][:-1]]
```

Comment:
```suggestion
                devname = self.disk_partition.device.split("/")[-1]
                disk_io = psutil.disk_io_counters(perdisk=True)[devname]
```

Get the correct device name at first, so you don't need that strange if. Or do I forgot some strange edge case?

Reply:
For me the partition names also work, e.g.: 

```
>>> import psutil
>>> psutil.disk_io_counters(perdisk=True)['nvme1n1p1'].read_count
469
>>> psutil.disk_io_counters(perdisk=True)['nvme1n1p2'].read_count
267601
>>> psutil.disk_io_counters(perdisk=True)['sda1'].read_count
5168
```

Why do you need device names, isn't partition names enough?

Reply:
Yeah, i had a brainfart... :D